src/poprox_storage/repositories/accounts.py

```diff
@@ -94,15 +94,17 @@ def store_account(self, account: Account) -> UUID | None:
         row = self.conn.execute(query).one_or_none()
         return row.account_id
 
-    def store_new_account(self, email: str, source: str) -> Account:
+    def store_new_account(self, email: str, source: str, sub_source: str = None) -> Account:
```

Comment:
I'd probably omit the underscore from subsource:
```suggestion
    def store_new_account(self, email: str, source: str, subsource: str = None) -> Account:
```